Docker/cpu/setup.jl

```diff
@@ -1,5 +1,6 @@
 #Pkg.add("TensorFlow")
-Pkg.clone("https://github.com/malmaud/TensorFlow.jl.git")
+using Pkg
+Pkg.add(PackageSpec(url="https://github.com/malmaud/TensorFlow.jl.git", rev = "master"))
```

Comment:
Now that TensorFlow.jl has stabilized, how about just `Pkg.add("TensorFlow", version="0.10")` instead of directly cloning `master`?